src/routes/login/LogIn.jsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import { login } from "../../redux/reducers/loginSlice";
+import { TextField } from "@material-ui/core";
+import { useFormik } from "formik";
+import { basicSchema } from "../../utilities";
+import { verifiedUser } from "../../redux/reducers/loginSlice";
+import { dispatch } from "../../redux/store/store";
+import "./LogIn.css";
+import { NavLink } from "react-router-dom";
+import Button from "../../components/button/Button";
+
+function LogIn() {
+  const formikData = useFormik({
+    initialValues: {
+      username: "",
+      password: "",
+    },
+    onSubmit: async (values, actions) => {
+      await new Promise((data) => setTimeout(data, 500));
+      try {
+        const user = await login(username, password);
+        dispatch(verifiedUser(user));
+      } catch (error) {}
+      actions.resetForm();
+    },
```

Comment:
???
why timer
why need async operation here
we just need to dispatch login action